packages/data-stores/src/plans/types.ts

```diff
@@ -156,8 +113,11 @@ export interface Feature {
 	type: string;
 }
 
+export type DetailsAPIFeature = Omit< Feature, 'type' >;
```

Comment:
Why is `Feature` an interface & `DetailsAPIFeature` a type?

Reply:
Resolving in favour of https://github.com/Automattic/wp-calypso/pull/49595#discussion_r574461555